### PR TITLE
RHCLOUD-21655: k8sCreate (POST) API behavior deviation from K8s server

### PR DESCRIPTION
**Pull request by florkbr**
See https://issues.redhat.com/browse/RHCLOUD-21655

### CODE REVIEW

File: packages/lib-utils/src/k8s/k8s-resource.ts [hunk 1]
```diff
@@ -62,7 +63,7 @@ export const k8sCreateResource = <
   fetchOptions = {},
 }: K8sResourceUpdateOptions<TResource>): Promise<TCreatedResource> =>
   commonFetchJSON.post<TCreatedResource>(
-    getK8sResourceURL(model, resource, queryOptions),
+    getK8sResourceURL(model, resource, omit(queryOptions, 'queryParams')),
```
Comment (andrewballantyne) [inline packages/lib-utils/src/k8s/k8s-resource.ts:66]: What is the URL out the other end -- because wouldn't the `resource` still make use of the name?

Comment (florkbr) [reply packages/lib-utils/src/k8s/k8s-resource.ts]: Yeah. I jumped the gun on opening a PR. I am working on addressing the name and workspace path params.

**Comment by florkbr** [comment]
Trying one more time to see if gateway timeouts resolve

**Pull request merged**